package/yast2-ruby-bindings.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Wed Mar  8 12:55:38 UTC 2017 - jreidinger@suse.com
+
+- Do not crash when invalid path is passed (bsc#1028081)
```

Comment:
This is good, but I am curious about the exact circumstances so I've come up with a more specific description:

> Do not crash when an invalid Yast::Path is constructed (one with a component starting or ending with a dash; bsc#1028081)